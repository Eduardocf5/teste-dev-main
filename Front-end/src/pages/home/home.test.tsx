import React from 'react';
import renderer from 'react-test-renderer';
import Home from '.';

test('render test', () => {
    const component = renderer.create(<Home />).toJSON();

    expect(component).toMatchSnapshot();
  });