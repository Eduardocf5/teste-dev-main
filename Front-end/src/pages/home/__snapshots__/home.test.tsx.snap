// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render test 1`] = `
<div>
  <div
    className="flex justify-end items-end flex-wrap mb-6 gap-6 text-sm"
  >
    <div>
      <span
        className="flex mb-2 ml-1 font-semibold"
      >
        Pesquisar por
      </span>
      <select
        className="rounded-lg p-2.5"
        id="filter"
        name="filter"
        onChange={[Function]}
        value="name"
      >
        <option
          value="name"
        >
          Nome
        </option>
        <option
          value="species"
        >
          Espécie
        </option>
        <option
          value="origin_name"
        >
          Nome de Origem
        </option>
        <option
          value="location_name"
        >
          Localização
        </option>
      </select>
    </div>
    <div>
      <input
        className="rounded-lg p-2"
        onChange={[Function]}
        placeholder="..."
        type="text"
        value=""
      />
    </div>
    <div>
      <span
        className="flex mb-2 ml-1 font-semibold"
      >
        Status
      </span>
      <select
        className="rounded-lg p-2.5"
        id="status"
        name="status"
        onChange={[Function]}
        value=""
      >
        <option
          value=""
        />
        <option
          value="Alive"
        >
          Vivo
        </option>
        <option
          value="Dead"
        >
          Morto
        </option>
        <option
          value="unknown"
        >
          Desconhecido
        </option>
      </select>
    </div>
    <div>
      <span
        className="flex mb-2 ml-1 font-semibold"
      >
        Gênero
      </span>
      <select
        className="rounded-lg p-2.5"
        id="gender"
        name="gender"
        onChange={[Function]}
        value=""
      >
        <option
          value=""
        />
        <option
          value="Female"
        >
          Feminino
        </option>
        <option
          value="Male"
        >
          Masculino
        </option>
        <option
          value="unknown"
        >
          Desconhecido
        </option>
      </select>
    </div>
    <div>
      <img
        alt="search"
        className="w-9 cursor-pointer"
        onClick={[Function]}
        src="search.svg"
      />
    </div>
  </div>
  <div
    className="grid xl:grid-cols-4 lg:grid-cols-3 md:grid-cols-2 gap-6"
  />
  <div
    className="flex flex-wrap justify-center gap-6 mt-6"
  >
    <div
      className="flex justify-center bg-white p-2 rounded-md gap-3 cursor-pointer min-w-40"
      onClick={[Function]}
    >
      <span>
        Mostrar Mais
      </span>
      <img
        alt="arrow-more"
        className="w-4 rotate-90"
        src="arrow.svg"
      />
    </div>
  </div>
</div>
`;
